app/package.json

```
@@ -57,6 +57,8 @@
     "@jupyterlab/terminal-extension": "~3.2.0",
     "@jupyterlab/theme-dark-extension": "~3.2.0",
     "@jupyterlab/theme-light-extension": "~3.2.0",
+    "@jupyterlab/toc": "~5.3.0",
+    "@jupyterlab/toc-extension": "~5.3.0-alpha.15",
```

We can now use the latest stable versions:

```suggestion
    "@jupyterlab/toc": "~5.2.3",
    "@jupyterlab/toc-extension": "~5.2.3",
```

---

Also these resolutions can be automatically generated with the `jlpm run integrity` script (after removing the previous `node_modules` to clean up)